### PR TITLE
feat: [5038] enable list view - add to dashboard

### PR DESCRIPTION
**Pull request by YounixM**
Summary:

Trace / Logs Explorer - List view - add to dashboard - should add a new panel of type list.

### CODE REVIEW

**Review by vikrantgupta25 — APPROVED** [review]

File: frontend/src/container/NewWidget/utils.ts [hunk 1]
```diff
@@ -189,50 +189,6 @@ export const panelTypeDataSourceFormValuesMap: Record<
 			},
 		},
 	},
-	[PANEL_TYPES.HISTOGRAM]: {
```
Comment (SagarRajput-7) [inline frontend/src/container/NewWidget/utils.ts:192]: Interesting. How was this not flagged by ESLint locally or in the PR checks?

**Review by SagarRajput-7 — DISMISSED** [review]

**Comment by makeavish** [comment]
@YounixM When I add to dashboard from logs page then the list takes longer to load, but same list from logs loads faster when I create new panel from scratch. And sometimes tab crashes: Screenshot 2024-06-18 at 17 13 29

Also faced 400 error at the end.

https://www.loom.com/share/a073ce3d7150416cb2ee225b05ae6ffc?sid=3d226ad3-8052-4444-9b14-ab44e3934304

**Comment by YounixM** [comment]
https://www.loom.com/share/703b72bc8ba241f1ad5929a06b90ced5?sid=4839e9ce-97f8-4581-a93a-c54bf3e1bb02

File: frontend/src/container/NewWidget/utils.ts (hunk 1 above)
Comment (YounixM) [reply frontend/src/container/NewWidget/utils.ts:192]: Have to check the rules on this.

**Comment by YounixM** [comment]
@makeavish : the mentioned issues should not happen now

**Review by SagarRajput-7 — APPROVED** [review]

**Review by vikrantgupta25 — APPROVED** [review]

**Comment by makeavish** [comment]
@YounixM Columns should be transferred to list view panel in dashboard: https://www.loom.com/share/f7250de5b8a749c6a9f74bd62a03f08a?sid=56cd116e-7795-4815-ae87-1b436d8ed027

**Pull request merged**